### DISCUSSION TITLE
How do I stop "Plugin Updates" from appearing?

### DISCUSSION THREAD

**Discussion by rstacruz** [LazyVim/LazyVim#3298] [Q&A] [answered]
Loading up nvim always shows a "Plugin Updates" popup similar to the one below. Is there a way to prevent this?

![image](https://github.com/LazyVim/LazyVim/assets/74385/39b0ce4e-8e88-4667-b711-55c67975712a)

Apologies if this isn't LazyVim related, I'm not actually sure which part of my setup does this.

**Comment by dpetka2001** [comment, marked as answer]
In the file `~/.config/nvim/lua/config/lazy.lua` find the line that says `checker = { enabled = true }` and change it to `checker = { enabled = true, notify = false }`. For more info you should check out the `lazy.nvim`'s [options](https://github.com/folke/lazy.nvim#%EF%B8%8F-configuration).

**Reply by rstacruz** [reply]
nice, this works!

**Reply by vtempest** [reply]
this should be the default. no need for annoying popup that gives erros mostly
